node-workspace/Dockerfile

```diff
@@ -3,19 +3,10 @@ FROM node:6.10
 MAINTAINER Sebastian Mandrean <sebastian@urb-it.com>
 
 # Environment variables
-ENV YARN_VERSION 0.21.3
 ENV PHANTOMJS_VERSION 2.1.14
 
 # Install dependencies
-RUN apt-get -y update \
- && apt-get -y install \
-	ocaml \
-	libelf-dev \
- && npm i -g \
-	yarn@$YARN_VERSION \
+RUN npm i -g \
 	phantomjs-prebuilt@$PHANTOMJS_VERSION
 
-# Clean up
-RUN rm -rf /var/cache/apt/archives
```

Comment:
I would still keep the cache cleanup since it's idiomatic Docker Alpine, but chain it to `npm -i` with `&&` to keep it in the same Docker image layer.